autode/bracket/base.py

```diff
@@ -0,0 +1,284 @@
+from typing import Union, Optional, TYPE_CHECKING
+from abc import ABC, abstractmethod
+
+from autode.values import Distance, GradientRMS
+from autode.bracket.imagepair import EuclideanImagePair
+from autode.methods import get_hmethod
+from autode.log import logger
+from autode import Config
+
+if TYPE_CHECKING:
+    from autode.species.species import Species
+    from autode.wrappers.methods import Method
+
+
+class BaseBracketMethod(ABC):
+    """
+    Base class for all bracketing methods
+    """
+
+    def __init__(
+        self,
+        initial_species: "Species",
+        final_species: "Species",
+        maxiter: int = 300,
+        dist_tol: Union[Distance, float] = Distance(1.0, "ang"),
```

Comment:
@t-young31 From what I can see from the tests, it can be reduced to as low as 0.5 angstrom before one image jumps over the barrier. However, the paper Kilmes et al., J. Phys.: Condens. Matter, 22 2010, 074203, say that the risk of jumping over is high if distance is below 1.0 A and they recommend switching to CI-NEB with 1 image at that point. So that's why I set that to 1.0 A. Do you think I should make it lower? Or make running CI-NEB the default?

Reply:
all good if that's the recommended value 👍🏼